crates/sui-framework/packages/dwallet-system/tests/tendermint_lc_tests.move

```diff
@@ -1,11 +1,80 @@
 #[test_only]
 module dwallet_system::lc_tests {
+
+    const SENDER: address = @0x012;
+
+    use dwallet::test_scenario;
+    // use dwallet::dynamic_field as fields;
+    use dwallet::test_utils::{Self};
+    use dwallet_system::tendermint_lc::{init_lc, Self};
+
     #[test]
     fun tendermint_state_proof_test() {}
     #[test]
-    fun tendermint_init_lc_test() {}
+    fun tendermint_init_lc_test() {
+        let scenario = test_scenario::begin(SENDER);
+        let height = 4;
+
+        let timestamp: vector<u8> = vector[1];
+        let next_validators_hash : vector<u8> = vector[2];
+        let root: vector<u8> = vector[3];
+        let ctx = test_scenario::ctx(&mut scenario);
+        let client = init_lc(height, timestamp, next_validators_hash, root, ctx);
+        
+        assert!(tendermint_lc::latest_height(&client) == height, 0);
+        test_scenario::end(scenario);
+        test_utils::destroy(client);
+    }
     #[test]
-    fun tendermint_verify_lc_test() {}
+    fun tendermint_verify_lc_test() {
+         let scenario = test_scenario::begin(SENDER);
+        let height = 6; 
+
+        let timestamp: vector<u8> = vector[50, 48, 50, 52, 45, 48, 55, 45, 49, 50, 84, 49, 49, 58, 48, 53, 58, 48, 57, 46, 51, 54, 54, 56, 52, 57, 49, 48, 52, 90];
```

Comment:
please add a comment where this data is coming from.

Reply:
Yes, for sure!